### PR TITLE
[MEN][94709274] display custom moving form for rent

### PR DESCRIPTION
**Pull request by michaelnease**
[story](https://www.pivotaltracker.com/story/show/94709274)

PR 2 of 3/4 for this story.

Hi @pasha @abouius @kevinolson I'm tagging you guys because I'm not sure if PRs in this repo are monitored.

This code takes the data-template value and passes it to the service.

### CODE REVIEW

**Comment by pasha** [comment]
:+1:

File: src/jquery-moving-service.coffee [hunk 1]
```diff
@@ -50,6 +50,7 @@ define [
           MovingTo_state: $(this).attr("data-state")
           MovingTo_city: $(this).attr("data-city")
           MovingTo_zip: $(this).attr("data-zip")
+          template: $(this).attr("data-template")
```
Comment (abouius) [inline src/jquery-moving-service.coffee]: It looks like this `template` parameter only gets added to the request URL if it has a value, is that accurate @michaelnease?  Just want to make sure as I just realized that your [server side changes](https://github.com/rentpath/service_moving_form/pull/27) might not handle an empty string value correctly.

Comment (pasha) [inline src/jquery-moving-service.coffee]: good catch

**Comment by michaelnease** [comment]
If a value isn't passed, then won't the delete method's default value be used? I have tested it without a value.

**Comment by abouius** [comment]
It depends on the value.  It if is an empty string, then that will be returned, since in Ruby an empty string evaluates as *truthey*

**Comment by michaelnease** [comment]
@pasha @abouius Can you take a look at this commit? I've reworked it. The template variable is no longer inside the form-params object and will not be passed if data-template doesn't exist. The service will need to be updated has well. I just wanted to run this by you guys first.

**Comment by abouius** [comment]
:+1:

**Comment by cleathers** [comment]
@michaelnease, @abouius glad to see y'all were able to add custom templates to this! I'm sure it cleaned up some code in the rent app :)

**Pull request merged**